graph-manager-docs/source/schema-reporting-protocol.mdx

```diff
@@ -0,0 +1,153 @@
+---
+title: Apollo schema reporting protocol
+sidebar_title: Schema reporting protocol
+description: Add automatic schema registration to your GraphQL server
+---
+
+This document specifies the protocol that any GraphQL server can implement to enable automatic [schema registration](./schema-registry/) with Apollo Graph Manager.
+
+A reference implementation of this protocol is included in [Apollo Server](https://github.com/apollographql/apollo-server/pull/4084/files).
+
+## Schema reporting endpoint
+
+Servers that implement the schema reporting protocol communicate with the following GraphQL endpoint:
+
+```
+https://edge-server-reporting.api.apollographql.com/api/graphql
+```
+
+## Protocol sequence
+
+The following sections illustrate the communication sequence between a GraphQL server (hereafter referred to as the **edge server**) and the schema reporting endpoint (**Apollo**):
+
+### Diagram
+
+```mermaid
```

Comment:
this diagram appears with overlapping text in the preview :( but maybe that shall be fixed soon

Reply:
It shall! https://github.com/apollographql/gatsby-theme-apollo/pull/123